### PR TITLE
Update dev meeting info

### PR DESCRIPTION
**Pull request by dcalvin**
### Related changes

- [ ] This change also requires further updates to the [website](https://github.com/chaos-mesh/website) (e.g. docs)
- [ ] This change also requires further updates to the `UI interface`
- Need to **cheery-pick to release branches**
  - [ ] release-2.3
  - [ ] release-2.2

### Checklist

CHANGELOG

- [ ] I have updated the `CHANGELOG.md`
- [x] I have labeled this PR with "no-need-update-changelog"

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] **Breaking backward compatibility**

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

**Review by Andrewmatilde — APPROVED** [review]
LGTM, PTAL @STRRL , @cwen0

**Review by STRRL — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -82,14 +82,17 @@ Please reach out for bugs, feature requests, and other issues via:
...
+  - Time: on the fourth Thursday of every month (unless otherwise specified)
+  - [RSVP here](https://community.cncf.io/chaos-mesh-community/)
+  - [Meeting minutes](https://docs.google.com/document/d/1H8IfmhIJiJ1ltg-XLjqR_P_RaMHUGrl1CzvHnKM_9Sc/edit?usp=sharing)
 
-**Quick links:**
-
-- [RSVP here](https://community.cncf.io/chaos-mesh-community/)
-- [Meeting notes](https://docs.google.com/document/d/1H8IfmhIJiJ1ltg-XLjqR_P_RaMHUGrl1CzvHnKM_9Sc/edit?usp=sharing)
+- Chaos Mesh Development Meeting (Releases, roadmap/features/RFC planning and discussion, issue triage/discussion, etc)
+  - Time: Every other Tuesday ((unless otherwise specified)
```
Comment (STRRL) [inline README.md:93]:
```suggestion
  - Time: Every other Tuesday (unless otherwise specified)
```

**Review by STRRL — COMMENTED** [review]
rest LGTM!

**Comment by STRRL** [comment]
waiting https://github.com/chaos-mesh/chaos-mesh/pull/3602 for fixing the CI "License checker"

**Pull request merged**